src/models/schemaversion.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (sequelize, DataTypes) => {
+  const SchemaVersion = sequelize.define(
```

Comment:
I think the file name should be `schemaVersion.js`

Reply:
Done